spec/spec_helper.rb

```diff
@@ -26,6 +26,12 @@
 VCR.configure do |config|
   config.cassette_library_dir = 'spec/support/fixtures/vcr_cassettes'
   config.filter_sensitive_data('<API_KEY>') { WorkOS.config.key }
+  config.filter_sensitive_data('<ACCESS_TOKEN>', :token) do |interaction|
+    JSON.parse(interaction.response.body)['access_token']
+  end
+  config.filter_sensitive_data('<REFRESH_TOKEN>', :token) do |interaction|
+    JSON.parse(interaction.response.body)['refresh_token']
+  end
```

Comment:
It'd be nice if this was the default and test authors didn't need to remember to opt-in, though I don't know the best way to implement that behavior without it also being inconsistent.

Reply:
ugh, i agree. i tried just having the handler work without tags but it errors out if you don't return something. (i guess i could return some bogus value?). this seemed to be the "vcr" way to handle this situation.